schemas/devContainerFeature.schema.json

```diff
@@ -0,0 +1,199 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "Feature": {
+      "additionalProperties": false,
+      "properties": {
+        "capAdd": {
```

Comment:
Could you add descriptions to each property? These will be shown in VS Code's (and presumably other editors') IntelliSense which helps users along without always having to go back and forth the editor and the documentation.